providers/marketo/metadata.go

```diff
@@ -28,14 +31,24 @@ func (c *Connector) ListObjectMetadata(ctx context.Context,
 	}
 
 	for _, obj := range objectNames {
-		// Constructing the request url.
 		url, err := c.constructMetadataURL(obj)
 		if err != nil {
 			return nil, err
 		}
 
-		resp, err := c.Client.Get(ctx, url.String())
+		httpResp, body, err := c.Client.HTTPClient.Get(ctx, url.String())
+		if err != nil {
+			logging.Logger(ctx).Info("failed to get metadata", "object", obj, "body", body, "err", err.Error())
```

Comment:
I'd use warning or error, not info

```suggestion
			logging.Logger(ctx).Error("failed to get metadata", "object", obj, "body", body, "err", err.Error())
```